foodsaving/groups/emails.py

```diff
@@ -69,6 +69,11 @@ def prepare_group_summary_data(group, from_date, to_date):
         'settings_url': settings_url,
     }
 
+    data['has_activity'] = any(data[field] > 0 for field in ['pickups_done_count', 'pickups_missed_count']) or \
+        any(len(data[field]) > 0 for field in ['feedbacks', 'messages', 'new_users'])
+
+    return data
```

Comment:
Returning None when there is no useful data seems more useful?

Reply:
I think it would be nice to have the stats written to influxdb even if no emails is sent. This was the easiest option, otherwise the calling function would need to do more work.

Reply:
I see I see. That is somewhat repurposing the intention of that stat (`karrot.email.group_summary`), it's for tracking emails that were sent (with their details), sending an email with 0 recipients is not really a thing. Is there anything in it that you cannot fetch from influxdb already?

Reply:
I think the only purpose is to check if the task actually ran, but there's already the other stat `group__send_summary_emails`. Ok, so it seems better to not send out the `karrot.email.group_summary` stat.

Reply:
One of the original purposes was to keep track of the point at which we send too much information in that email. At the moment it shows _all_ the messages/feedback.

Another purpose is to have a way to assess if the logic is correct (e.g. no sudden leap in the amount of things matched), so we can be slightly better positioned to detect that.

It also might be nice to know how many people are being sent the weekly notifications.

Checking if the task ran is not the purpose of that stat.

Reply:
> no sudden leap in the amount of things matched

Wouldn't it be helpful to see when no things matched (instead of no data point)?

Reply:
Ok, lets go for it.